webpack.config.js

```diff
@@ -352,6 +353,9 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 				} ),
 			shouldEmitStats && new webpack.ProgressPlugin( createProgressHandler() ),
 			new BuildCustomPropertiesCssPlugin(),
+			new MomentTimezoneDataPlugin( {
```

Comment:
we may want to make this configurable for sdk builds.

Reply:
Would an environment variable be the way to go here? E.g. `STRIP_OLD_TIMEZONE_DATA`?

Reply:
Or would it be better to tie this to an existing environment variable (and if so, which one?)

Reply:
@Automattic/calypso-sdk any opinions here?

I think `STRIP_OLD_TIMEZONES` as an env var should be fine for now. We can always change it later.

Reply:
I believe our Gutenberg extensions not having any dates currently or rely on external moment shipped in WP core, so this shouldn't affect us in any way.

Reply:
I added the environment variable but didn't modify the NPM scripts. This means that by default it *will not* be used.

Let me know if I should set it in the NPM scripts or `package.json` somewhere, or if instead these files should remain clean and it should be enabled in the build server somehow.